pages/partners_new.tsx

```diff
@@ -1,15 +1,252 @@
+  {
+    title: 'Nonprofits',
+    description:
+      'We strive to bring enterprise-level operational maturity to nonprofits in need.',
+    icon: <Code style={{ color: designColor.white }} />,
+  },
+  {
+    title: 'Governments',
+    description:
+      'Local governmental organizations aimed at building a better community deserve a helping hand.',
+    icon: <Storage style={{ color: designColor.white }} />,
+  },
+  {
+    title: 'Open Source',
+    description:
+      'Whenever possible, we will cooperate using open source assets that can be leveraged by anyone.',
+    icon: <DataObject style={{ color: designColor.white }} />,
+  },
+  {
+    title: 'Maintainability',
+    description:
+      "When we build, we don't just build for today. Our services and tools are designed for long-term usability and scale.",
+    icon: <DataObject style={{ color: designColor.white }} />,
+  },
+]
+
 const PartnersPage = () => {
   const theme = useTheme()
+  const palette = theme.palette
 
   return (
-    <>
-      <Typography sx={{ color: theme.palette.primary.main }}>
-        This is the new Partner page for you to fill.
-      </Typography>
-    </>
+    <div>
+      <Container
+        sx={{
+          backgroundColor: theme.palette.primary.main,
+          alignItems: 'center',
+          display: 'flex',
+          flexDirection: 'column',
+          paddingY: '5rem',
+        }}
+        maxWidth={false}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', md: 'row' },
+            alignItems: 'center',
+          }}
+        >
+          <Box sx={{ width: '50%', textAlign: 'left' }} maxWidth={'422px'}>
+            <Typography
+              variant="displayLarge"
+              sx={{
+                color: theme.palette.primary.contrastText,
+                marginBottom: '2rem',
+              }}
+            >
+              Partner with us
+            </Typography>
+            <Typography
+              variant="headlineLarge"
+              sx={{ color: theme.palette.primary.contrastText }}
+            >
+              Open Seattle works with Seattle-based nonprofits to create
+              customized digital solutions for their needs, for free.
+            </Typography>
+            <div style={{ margin: '2rem' }}>
+              <Link href="/partner-application" passHref>
+                <Button variant="contained" color="secondary">
+                  Partner with us
+                </Button>
+              </Link>
+            </div>
+          </Box>
+          <Box sx={{ width: '50%' }} maxWidth={'700px'}>
+            <img
+              src={PartnerImage.src}
+              alt="Partner page graphic"
+              style={{ objectFit: 'cover' }}
+            ></img>
+          </Box>
+        </Box>
+      </Container>
+      <Container
+        style={{
+          backgroundColor: theme.palette.background.default,
+          color: designColor.black,
```

Comment:
nit: we prefer to get the colors from the theme object instead of the designColor one, in case we want to enable a dark mode in the future. so you could use `theme.palette.primary.main` for the dark text color